backend/lambdas/api/system_services/system_services/util/service.py

```diff
@@ -83,14 +83,18 @@ def get_services(cls, email: str, services_dict: dict):
 
     def _test_auth(self):
         key = get_api_key(self._service["secret_loc"])
+        bitbucket_v1 = "/1.0" in self._service["url"] and self._service["type"] == ServiceType.BITBUCKET
+
         if key is None:
             self._error = "Unable to retrieve key"
         if self._service["type"] == ServiceType.GITHUB:
             self._test_github(key)
         elif self._service["type"] == ServiceType.GITLAB:
             self._test_gitlab(key)
-        elif self._service["type"] == ServiceType.BITBUCKET:
-            self._test_bitbucket(key)
+        elif self._service["type"] == ServiceType.BITBUCKET and not bitbucket_v1:
```

Comment:
Would it make sense to split `BITBUCKET` into two separate service types? If the APIs vary enough that we have to differentiate in code, IMO they are two different types of services. Some shared functionality can still be abstracted to shared functions where possible.

Reply:
I agree, they are different services. I'm okay with splitting them up within the services API. Because doing this throughout Artemis would be a huge lift. What do you think?

Reply:
Yeah, I think we should split them here for now, including defining them separately in that Enum you linked in your comment, but limit the scope to this at this time.